wger/core/static/js/wger-core.js

```diff
@@ -119,6 +119,20 @@ function setProfileField(field, newValue) {
   });
 }
 
+function changeWorkoutType (event) {
+
+  if (event.value === '1') {
+    $('#id_sets').val(4);
+    $('#id_sets_value').html($('#id_sets').val());
+    updateAllExerciseFormset();
+  } else {
+    $('#id_sets').val(1);
+    $('#id_sets_value').html($('#id_sets').val());
+    updateAllExerciseFormset();
```

Comment:
Dont repeat yourself here. Move the bottom 2 lines of code in both if-else conditions and place it at the bottom. They will always be called in both cases.